torch/distributed/_tensor/device_mesh.py

```diff
@@ -66,6 +69,22 @@ def create_child_mesh(
     def get_parent_mesh(self, device_mesh: "DeviceMesh") -> Optional["DeviceMesh"]:
         return self.child_to_parent_mapping.get(device_mesh, None)
 
+    def get_parent_mesh_dim(self, device_mesh: "DeviceMesh") -> Optional[int]:
```

Comment:
Is there any reason why we want to expose this to users? I mean when users type in `['TP', 'DP']`, they already know the order? If this is mostly for state_dict, maybe we want to make it a private API?

Reply:
This is not exposed to the user. All these helper API sits in _MeshEnv instead of DeviceMesh, and the _MeshEnv class is a private class. We kept all the helper APIs that we need in _MeshEnv so DeviceMesh can be cleaner.

Reply:
I see..